File: official/docs/java/v6/batches/scan-forms.java
package batches;

import com.easypost.exception.EasyPostException;
import com.easypost.model.Batch;
import com.easypost.service.EasyPostClient;

public class ScanForms {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        Batch batch = client.batch.createScanForm("batch_...");

        System.out.println(batch);
    }
}

File: official/docs/node/v6/batches/scan-forms.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const batch = await client.Batch.retrieve('batch_...');

  const batchWithScanForm = await client.Batch.createScanForm(batch.id);

  console.log(batchWithScanForm);
})();
